pkg/playground/react-demo-listing.jsx

```diff
@@ -51,6 +51,13 @@
             window.alert("navigated to item: " + msg);
         };
 
+        var handleAddClick = function(event) {
+            if (event.button !== 0)
+                return;
+
+            window.alert('this link could add a row');
```

Comment:
I propose changing this to `This link could open a dialog to create a new entry` in order to avoid confusion. Otherwise I would understand that this link adds an entry without opening anything - for such an action I would expect an actual button though, not a link.